tests/views_integration/Tribe/Events/Views/V2/HTML/MonthTest.php

```diff
@@ -90,10 +90,11 @@ protected function given_month_data() {
 			array_merge(
 				$this->template->get_global_values(),
 				[
-					'today_date'          => '2019-07-03',
-					'grid_date'           => '2019-07-03',
-					'formatted_grid_date' => '2019-07-03',
-					'days'                => $days
+					'today_date'                 => '2019-07-03',
+					'grid_date'                  => '2019-07-03',
+					'formatted_grid_date'        => '2019-07-03',
+					'formatted_grid_date_mobile' => '7/3/2019',
+					'days'                       => $days
```

Comment:
Not essential, so feel free to skip, but generally we want the last element in the array to have a trailing comma.